public/models/contribution-month/contribution-month.js

```diff
@@ -301,15 +301,31 @@ var ContributionMonth = DefineMap.extend("ContributionMonth", { seal: false }, {
 	}
 });
 
+const monthsUntilDecay = 6;
+const minimumPoints = 1;
+const oneMonth = 1000 * 60 * 60 * 24 * 30;
```

Comment:
I prefer to use a 4-week period rather than a "month" since it is a little more consistent.

Reply:
That would make 13 months a year so contributions would decay faster than intended. As I understand it, the goal is to make points decay twice a year. I am merging for now, but I am open to further discussion about this implementation. I can be changed if needed.